README.md

```diff
@@ -3,3 +3,13 @@
 `nectar help` to see the available commands.
 
 Check `./sample-config.toml` if you want to customize the config.
+
+## Security advisory
+
+-   The seed file is used to generate the Bitcoin and Ethereum wallets.
+-   Bitcoin funds are held in a new wallet generated in the bitcoind instance. Keep your bitcoind instance secure. 
```

Comment:
Two sentences on a single line, also there is whitespace at the end of this line.